src/kirby/components/list/list.component.scss

```diff
@@ -79,6 +79,32 @@ $shadow: get-elevation(2);
   }
 }
 
+:host(.no-shape) {
+  ion-list,
+  ion-item-sliding {
+    border: none;
+    box-shadow: none;
+    background: none;
+
+    & > ion-item {
+      background: none;
+      --padding-start: 0;
+      --inner-padding-end: 0;
+      padding: 0;
+    }
+  }
+}
+
+:host(.spaced) {
+  ion-list > ion-item {
+    padding-bottom: size('s');
```

Comment:
Please use [Ionics css variables](https://ionicframework.com/docs/api/item#css-custom-properties) instead for border, padding etc.